### PR TITLE
Generate dash component cli

### PR DESCRIPTION
**Pull request by T4rk1n**
Change the `build:py` command to use the combined `extract-meta.js` and class file generation.

### CODE REVIEW

File: {{cookiecutter.project_shortname}}/tests/test_usage.py [hunk 1]
```diff
@@ -6,5 +9,19 @@ def test_render_component(dash_app, selenium):
...
-    my_component = selenium.find_element_by_id('input')
+    # Get the generated component input with selenium
+    # The html input will be a children of the #input dash component
+    my_component = selenium.find_element_by_css_selector('#input > input')
+
+    # Send keys to the custom input.
+    my_component.send_keys('Hello dash')
+
+    # Sleep for the output callback to complete.
+    time.sleep(1)
```
Comment (rmarren1) [inline {{cookiecutter.project_shortname}}/tests/test_usage.py]: Does the expected conditions wait work here? This test might be flaky.

**Comment by T4rk1n** [comment]
> Also, it looks like there is a lot of code repeated in the circleci config. Can there be a template for all steps excluding the last, a template for the 2.7 and 3.6 tests, then an extra step for the 3.7 build + test, then mix + match to 🐫

Can't override only parts of steps so they have to be repeated. The 3.7 step build the components externally to run `test/test_usage.py`, I couldn't test the actual `build:all` command in the generation tests so I thought about this one. Both tests perform project generation & build, just different build method. Not sure we need to duplicate the tests on all python versions, as long as each one is tested in a way.

**Comment by rmarren1** [comment]
Oh ok, did not know YAML doesn't allow merging lists :cry:.
Looks good after the changes in `dash`

**Comment by rmarren1** [comment]
:dancer:

**Pull request merged**